### PR TITLE
String array bug fixed

### PR DESCRIPTION
**Pull request by npatrik11**

### CODE REVIEW

**Comment by Annosz** [comment]
Please, next time provide some description of your work.

**Comment by Annosz** [comment]
Next time be carefull not to push .class files!

File: src/main/java/org/json/simple/ItemList.java [hunk 1]
```diff
@@ -40,7 +41,9 @@ public List getItems(){
 	}
 	
 	public String[] getArray(){
-		return (String[])this.items.toArray();
+		Object[] objectArray = this.items.toArray();
+		String[] stringArray = Arrays.copyOf(objectArray, objectArray.length, String[].class);
```
Comment (Annosz) [inline src/main/java/org/json/simple/ItemList.java]: I think it's not necessary to save the object array to a separate variebla, we can return with it right away.

**Review by Annosz — CHANGES_REQUESTED** [review]

**Comment by npatrik11** [comment]
I removed the unnecessary local variable.

**Review by Annosz — APPROVED** [review]

**Pull request merged**